airflow/models/importerror.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe this file should be called `exception.py` and contain all the exceptions associated with model?

Reply:
`exception.py` makes me think of Python exceptions, and `airflow/exceptions.py` contains Python exceptions. How about `errors.py` and we could group errors/failures in there? Just a thought.

Reply:
It's OK for me.